requirements.txt

```diff
@@ -27,7 +27,7 @@ plaster==1.0
 plaster-pastedeploy==0.7
 PyBrowserID==0.14.0
 pycparser==2.19
-PyFxA==0.7.6
+PyFxA==0.7.7
```

Comment:
Updating to the new version seems to have allowed `make test` to run successfully for me.